app/assets/stylesheets/govuk_publishing_components/components/_chat-entry.scss

```diff
@@ -0,0 +1,27 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.gem-c-chat-entry {
+  display: flex;
+  align-items: flex-start;
+
+  &.gem-c-chat-entry--border-top {
+    border-top: 2px solid govuk-colour("blue");
+    padding-top: govuk-spacing(5);
+  }
+
+  &.gem-c-chat-entry--border-bottom {
+    border-bottom: 2px solid govuk-colour("blue");
+    padding-bottom: govuk-spacing(5);
+  }
+
+  .gem-c-chat-entry__image {
+    flex-shrink: 0;
+    margin-right: govuk-spacing(3);
+  }
+
+  .gem-c-chat-entry__heading {
+    @include govuk-typography-common;
+    @include govuk-font-size(24);
+    margin: 0 0 govuk-spacing(2);
+  }
```

Comment:
From my understanding of the styleguide, these should not be nested and be positioned at the top level with `.gem-c-chat-entry`.

Reply:
@dnkrj could you please share a link to the style notes? I noticed some comments about [avoiding deep nesting](https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component_conventions.md#bem), but I see a variety of approaches to nesting in other component styles (which might be incorrect).

Reply:
I think that's the guidance you've found there. Generally the use of BEM means we shouldn't have to nest element or modifier classes like these (although of course there are exceptions).

Reply:
Thanks both. I've removed the nesting now